AEPAnalytics/Sources/Analytics.swift

```diff
@@ -310,6 +317,17 @@ public class Analytics: NSObject, Extension {
         }
     }
 
+    /// Processes Reset identites event
+    /// - Parameter:
+    ///   - event: The Reset identities event
+    private func handleResetIdentitiesEvent(_ event: Event) {
+        Log.debug(label: LOG_TAG, "\(#function) - Resetting all Identifiers")
+        analyticsDatabase?.reset()
+        analyticsState.resetIdentities()
+        analyticsProperties.reset()
```

Comment:
AnalyticsProperties#reset seems to also reset the locale, that may not be required for resetIdentities.

Reply:
@emdobrin I don't see AnalyticsProperties#locale being used anywhere. We get the locale from serviceProvider directly and don't see anywhere we persist it in AnalyticsProperties. https://github.com/adobe/aepsdk-analytics-ios/blob/b9121586e787a3700fbaba319304b34df2e7aab9/AEPAnalytics/Sources/Analytics.swift#L444